lammps-tutorials.tex

```diff
@@ -613,48 +577,38 @@ \subsubsection{My first input}
 avoid so-called ``close contacts'' between atoms which can cause
 problems due to causing large forces.
 
-\begin{figure}
-\centering
-\includegraphics[width=0.55\linewidth]{LJ}
-\caption{The binary mixture simulated during \hyperref[lennard-jones-label]{Tutorial 1}.
-  The atoms of type 1 are represented as small red spheres, the atoms of type 2 as large
-  green spheres, and the edges of the simulation box are represented a blue sticks}
-\label{fig:LJ}
-\end{figure}
-
 \paragraph{Settings}
 Now we provide the settings for the two types of atoms.  Modify the
-\textit{Settings} category of \textit{input.lmp} as follows:
+\textit{Settings} category of \textit{initial.lmp} as follows:
 {\normalsize
 \begin{verbatim}
- # 3) Settings
- mass 1 1.0
- mass 2 1.0
- pair_style lj/cut 2.5
- pair_coeff 1 1 1.0 1.0
- pair_coeff 2 2 0.5 3.0
+# 3) Settings
+mass 1 1.0
+mass 2 10.0
+pair_style lj/cut 4.0
+pair_coeff 1 1 1.0 1.0
+pair_coeff 2 2 0.5 3.0
 \end{verbatim}
 }
 
-The two \textit{mass} commands assign a mass of 1.0 mass unit
-to both atoms of type 1 and 2, respectively.  The same effect could be
-achieved with a single \textit{mass * 1.0} command. The star (*) character
-is a wild card representing all atom types.
+The two \textit{mass} commands assign a mass of 1.0 and 10.0 mass units
+to the atoms of type 1 and 2, respectively.
 
-The third line, \textit{pair\_style lj/cut 2.5}, requests that the atoms will
-be interacting through a Lennard-Jones (LJ) potential with a cut-off equal to
-$r_c = 2.5$ length units \cite{wang2020lennard,fischer2023history}:
+The third line, \textit{pair\_style lj/cut 4.0}, requests that the atoms
+will be interacting through a Lennard-Jones (LJ) potential with a
+cut-off equal to $r_c = 4.0$ length units
+\cite{wang2020lennard,fischer2023history}:
 $$E_{ij} (r) = 4 \epsilon_{ij} \left[ \left( \dfrac{\sigma_{ij}}{r} \right)^{12}
   - \left( \dfrac{\sigma_{ij}}{r} \right)^{6} \right], ~ \text{for} ~ r
 < r_c,$$ where $r$ is the inter-particle distance, $\epsilon_{ij}$ is
 the depth of the potential well that sets the interaction strength, and
 $\sigma_{ij}$ represents the distance where the potential energy of the
-LJ potential is zero.  Here, the indexes \textit{ij} refer to the
+LJ potential is zero.  Here, the indexes \textit{ij} refer to pairs of
 particle types \textit{i} and \textit{j}.  The minimum of the LJ
 potential in this formulation is at
 $\bar{\sigma} = 2^{\frac{1}{6}}\sigma$.  Sometimes people use an
 alternate formulation of the LJ potential which uses $\bar{\sigma}$,
-i.e. the minimum of the potential.  By substituting
+i.e.~the minimum of the potential.  By substituting
 $\sigma = \frac{\bar{\sigma}}{2^{\frac{1}{6}}}$ in the formulation above
 we get the alternate but equivalent expression:
 $$E_{ij} (r) = \epsilon_{ij} \left[ \left( \dfrac{\bar{\sigma}_{ij}}{r} \right)^{12}
```

Comment:
I am not convinced that the part about the alternate formulation of the LJ potential is necessary for absolute LAMMPS beginner facing their first tutorial (unless you've noticed that this particular confusion was commonplace?). In the website, I place such "extra" information in color boxes that users can skip if they don't want the info, this piece of information could definitely go there, but for the articles I think we could go "straight to business". 

edit: may be to be placed in a "\begin{note}" block?

After reading the "pair_style lj/cut" page, I saw a few typo. I remmeber that you sometimes collect "typo" from the Matsi forum in the past and fix them yourself, but to give you less work, can I simply create a PR with such minor corrections on the lammps github? (I am aware that the LAMMPS documentation does not encourage individual PR for minor correction though : "If you find an error or omission in the LAMMPS documentation or on the LAMMPS website, or have a suggestion or content for something to clarify or include, send an email to the developers." from https://www.lammps.org/contribute.html)

Reply:
> I am not convinced that the part about the alternate formulation of the LJ potential is necessary for absolute LAMMPS beginner facing their first tutorial

I am tempted to mention this, because the question of which pair style to use for the alternate form is a recurring one. I just noticed that the same information, perhaps a bit more terse is in the lj/cut pair style documentation at a rather prominent place. So it should be reduced to a short statement, perhaps in a "note" box. It is particularly beginners, that confuse there. So it cannot be mentioned enough. At the same time, expanding the LAMMPS manual and just pointing to it, may be just as well.

> can I simply create a PR with such minor corrections on the lammps github
You can submit a "collection" PR like I often do. Just call it a collection of small fixes for the manual and submit it as Draft. ... or just add review suggestions like you are doing here to my current collection (which is growing beyond proportion due to the LAMMPS-GUI changes and the fact that nobody is approving it, so I am adding to it and not merging it. I expect more updates for the manual soon because at the developer meeting tomorrow, I would like to organize a general pre-stable documentation check across the developers where we may find more things. We're still about 10 days away from the planned release day.